src/Workleap.OpenApi.MSBuild/Spectral/SpectralRulesetManager.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Microsoft.Build.Framework;
+using YamlDotNet.RepresentationModel;
+
+namespace Workleap.OpenApi.MSBuild.Spectral;
+
+internal sealed class SpectralRulesetManager
+{
+    private const string WorkleapRulesetVersion = "0.9.0";
+    private const string WorkleapRulesetDownloadUrlFormat = "https://raw.githubusercontent.com/gsoft-inc/wl-api-guidelines/{0}/.spectral.{1}.yaml";
```

Comment:
should this be {0} or main? given that folks will be using tagged versions.

Reply:
{0} since here it target a specific profile (like backend or frontend), so we want to be driven by the version of the library to prevent targetting latest.